x/wasm/keeper/test_common.go

```diff
@@ -56,7 +56,9 @@ func CreateTestInput(t *testing.T, tempDir string) (sdk.Context, auth.AccountKee
 	isCheckTx := false
 	ctx := sdk.NewContext(ms, abci.Header{}, isCheckTx, log.NewNopLogger())
 	cdc := MakeTestCodec()
-	appCodec := codecstd.NewAppCodec(cdc)
+	interfaceRegistry := codectypes.NewInterfaceRegistry()
```

Comment:
Should we just pull in the `MakeCodec` from `app` to this file and use it?

It seems it did a number of register commands that might be needed to some infrastructure later. I did spend ~2 hours hunting down a bug that was caused by not registering some amino codec in my test code.

Reply:
good idea but this creates cyclic dependencies:
```
package github.com/cosmwasm/wasmd/app
	imports github.com/cosmwasm/wasmd/x/wasm
	imports github.com/cosmwasm/wasmd/x/wasm/client/cli
	imports github.com/cosmwasm/wasmd/x/wasm/keeper
	imports github.com/cosmwasm/wasmd/app
```

Reply:
I actually meant cut-and-paste that code :grimacing: 

Too many circular deps in the sdk. Anyway, good to merge as is, or with this cut and paste.